packages/handlers/odata/yaml-config.graphql

```diff
@@ -0,0 +1,16 @@
+extend type Handler {
+  oData: ODataHandler
```

Comment:
Add docs :) To all fields. 

Also, now we have website, so add it there as well, and update the list of available handlers.

Reply:
Done!